erpnext/hr/doctype/attendance_request/attendance_request.json

```
@@ -115,11 +110,28 @@
    "options": "Attendance Request",
    "print_hide": 1,
    "read_only": 1
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Status",
+   "options": "Attendance Status",
+   "reqd": 1
+  },
+  {
+   "depends_on": "half_day",
+   "fieldname": "remaining_half_day_status",
+   "fieldtype": "Link",
+   "label": "Remaining Half Day status",
```

```suggestion
   "label": "Remaining Half Day Status",
```
Title casing for labels